<launch>

<include file="$(find realsense2_camera)/launch/rs_camera.launch"/>

<node pkg="nimo_perception" name="StalkDetecty" type="StalkDetect.py" output="screen"/>

</launch>